databases/cassandra/cassandra.go

```diff
@@ -111,6 +116,10 @@ func Dial(conf *Config) (*Database, error) {
 // already been initialized this function will not execute but return with a nil error. If any issues occur
 // while initializing an error will be returned.
 func (db *Database) Initialize() error {
+	//
+	if db == nil {
+		return fmt.Errorf("%s , db = %v", dbNill, db)
+	}
```

Comment:
I'm trying to understand how we could call `Initialize()` part of the DB interface with a nil db?

Reply:
i am not sure why , but it was getting nil when i was running the test cases ,

it was not happening if i was running the application

Reply:
Interesting, but that sounds more like a problem in the tests area than in the DB `Initialize()` function. Let's add our nil checks there.